src/main/java/seedu/address/logic/AnakinLogicManager.java

```diff
@@ -6,43 +6,43 @@
 
 import seedu.address.commons.core.ComponentManager;
 import seedu.address.commons.core.LogsCenter;
-import seedu.address.logic.Anakin_commands.Anakin_Command;
-import seedu.address.logic.Anakinparser.AnakinParser;
+import seedu.address.logic.AnakinCommands.AnakinCommand;
+import seedu.address.logic.AnakinParser.AnakinParser;
 import seedu.address.logic.commands.CommandResult;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.Anakin_Model;
-import seedu.address.model.Anakin_deck.Anakin_Deck;
+import seedu.address.model.AnakinDeck.AnakinDeck;
+import seedu.address.model.AnakinModel;
 
 /**
  * The main LogicManager of the app. Implements AnakinLogic
  */
 public class AnakinLogicManager extends ComponentManager implements AnakinLogic {
     private final Logger logger = LogsCenter.getLogger(AnakinLogicManager.class);
 
-    private final Anakin_Model model;
+    private final AnakinModel model;
     private final CommandHistory history;
-    private final AnakinParser anakinParser;
+    private final AnakinParser AnakinParser;
 
-    public AnakinLogicManager(Anakin_Model model) {
+    public AnakinLogicManager(AnakinModel model) {
         this.model = model;
         history = new CommandHistory();
-        anakinParser = new AnakinParser();
+        AnakinParser = new AnakinParser();
```

Comment:
Should this be AnakinParser or anakinParser?

Reply:
AnakinParser